src/SDKs/ApplicationInsights/ApplicationInsights.Tests/ApplicationInsights.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <Description>ApplicationInsights.Tests Class library</Description>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452;</TargetFrameworks>
```

Comment:
Every SDK test should be targeting netCore 1.1, any reason this needs to be target .NET 452?

Reply:
updated. thanks!